net-misc/urlscan/urlscan-0.9.1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Permit to launch URLs to webrowser inside Mutt mail client."
+HOMEPAGE="https://github.com/firecat53/urlscan"
+SRC_URI="https://github.com/firecat53/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="dev-python/urwid[${PYTHON_USEDEP}]"
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]
+${RDEPEND}"
+
+DOCS=( COPYING README.rst )
+
+python_prepare_all () {
```

Comment:
Well, this works, but if you want to reduce text it can be done in `src_prepare` too.